### PR TITLE
Remove ksonnet from the Cloud Filestore instructions.

### LINKED ISSUE #842: Kubeflow Filestore documentation is out dated
see https://github.com/kubeflow/pipelines/issues/1564 for original issue. 

Documentation instructions refer to using KS.. this can be simply done with Kubectl and directly deploying yamls for pv / and pvc.

Documentation link: https://www.kubeflow.org/docs/gke/cloud-filestore/

Example for out dated documentation :
cd //${KFAPP}/ks_app
ks generate google-cloud-filestore-pv google-cloud-filestore-pv --name="kubeflow-gcfs" 
--storageCapacity="${GCFS_STORAGE}" 
--serverIP="${GCFS_INSTANCE_IP_ADDRESS}"

### PR DESCRIPTION
**Pull request by jlewi**
* We can just link to the instructions on the GKE site which describes how
  to create PVs and PVC to access the volume.

Fix #842

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1136)

### CODE REVIEW

**Comment by jlewi** [comment]
Preview
https://deploy-preview-1136--competent-brattain-de2d6d.netlify.com/docs/gke/cloud-filestore/

/assign @abhi-g

File: content/docs/gke/cloud-filestore.md [hunk 1]
```diff
@@ -88,28 +88,7 @@ mykubeflow-nfs  us-east1-d  STANDARD  1024         kubeflow         10.20.148.19
...
-     gcloud --project=${PROJECT} beta filestore instances list
-     ```
-
-Apply the changes:
-
-```
-cd /<path-to-kubeflow-deployment>/${KFAPP}
-kfctl apply k8s
-```
+Now follow the instructions [Accessing Fileshared from Google Kubernetes](https://cloud.google.com/filestore/docs/accessing-fileshares) to create a PV and PVC.
```
Comment (sarahmaddox) [inline content/docs/gke/cloud-filestore.md]: s/Fileshared/Fileshares

**Comment by sarahmaddox** [comment]
/assign
/lgtm
/approve

Just one typo to fix. :)

File: content/docs/gke/cloud-filestore.md (hunk 1 above)
Comment (jlewi) [reply content/docs/gke/cloud-filestore.md]: Thanks. @sarahmaddox PTAL

**Comment by sarahmaddox** [comment]
Thanks!

/hold cancel
/lgtm

**Pull request merged**